examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -9,13 +9,13 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model_name = 'sasrec'  # 'sasrec' or 'ssept'
</pre><p>Minor detail, if we add <code>model_name</code> to the previous cell, which is the one that is configured with papermill,  we could add an integration test for SSEPT, just copying this one <a href="https://github.com/microsoft/recommenders/blob/main/tests/integration/examples/test_notebooks_gpu.py#L622" target="_blank">recommenders/test_notebooks_gpu.py at main · microsoft/recommenders (github.com)</a> and updating the outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1678/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Modified the test to have two parameters, one for SASRec and one for SSEPT.